lib/actions.ts

```diff
@@ -1,15 +1,28 @@
 "use server";
 
-import * as db from "@/lib/db";
-import * as Schemas from "drizzle/schema";
-import { auth } from "./auth";
-import * as blob from "@vercel/blob";
-import { NewProjectFormSchema } from "./types";
-import { redirect } from "next/navigation";
-import { defaulTaskSpecs } from "./tasks";
+import * as db from '@/lib/db';
+import * as Schemas from 'drizzle/schema';
+import { signIn } from '@/lib/auth';
+import { auth } from './auth';
+import * as blob  from '@vercel/blob';
+import { NewProjectFormSchema } from './types';
+import { redirect } from 'next/navigation';
+import { defaulTaskSpecs } from './tasks';
 
 const VERCEL_BLOB_FAKE_FILES = true;
 
+export async function createUserFromRegister(data: any) {
```

Comment:
A defined type rather than any is much better here

Reply:
Updated!